transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -78,6 +79,9 @@ public Integer call() throws Exception {
     private volatile int wakenUp;
     private volatile int ioRatio = 50;
 
+    // See http://man7.org/linux/man-pages/man2/timerfd_create.2.html.
+    static final long MAX_SCHEDULED_DAYS = TimeUnit.SECONDS.toNanos(999999999) + TimeUnit.NANOSECONDS.toSeconds(999999999);
```

Comment:
`TimeUnit.NANOSECONDS.toSeconds(999999999)`

That evaluates to zero.  Is this calculation correct?

Reply:
we can just remove the `TimeUnit.NANOSECONDS.toSeconds(999999999)` here and keep it simple. Will do

Reply:
So if the variable is named `MAX_SCHEDULED_DAYS`   why are there no "day" like constants in the arithmetic?  It's kind of odd to put a "seconds" value into a "days" variable.

Reply:
carl because I am stupid... I forgot to push the latest change :(